apps/condo/domains/acquiring/tasks/chargeRecurrentPayments.spec.js

```diff
@@ -114,7 +115,7 @@ describe('charge-recurrent-payments', () => {
             const notification = await Message.getOne(adminContext, {
                 type: RECURRENT_PAYMENT_PROCEEDING_SUCCESS_RESULT_MESSAGE_TYPE,
                 uniqKey: `rp_${recurrentPayment.id}_1_true`,
-            })
+            }, MESSAGE_FIELDS)
```

Comment:
why aren't test utilities used here?

Reply:
and below in this file

Reply:
and in other tests

Reply:
I guess because of specs, idk

Reply:
I tried to change it to `testUtils` – it works fine.
Don't see any reasons to use it here and in other same places, but now I'll leave it as it is.